src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -5,10 +5,12 @@
 using System.Threading.Tasks;
 using System.Reflection;
 
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

using System.Threading.Tasks;
using System.Reflection;
```
This should be in the top of the file